include/seastar/core/std-coroutine.hh

```diff
@@ -21,8 +21,8 @@
 
 #pragma once
 
-// Clang < 15 only supports the TS
-#if __has_include(<coroutine>) && (!defined(__clang__) || __clang_major__ >= 15)
+// Clang < 14 only supports the TS
```

Comment:
isn't "only" incorrect for clang 14?

Reply:
In what way?

Clang 12 only supports the TS
Clang 13 only supports the TS
Clang 14 supports the TS and c++20 coroutines
Clang 15 supports c++20 coroutines (TS removed)

Reply:
Thinko, comment is correct.